sql/core/src/main/scala/org/apache/spark/sql/execution/debug/package.scala

```diff
@@ -73,14 +74,14 @@ package object debug {
    * @param plan the query plan for codegen
    * @return single String containing all WholeStageCodegen subtrees and corresponding codegen
    */
-  def codegenString(plan: SparkPlan): String = {
+  def codegenString(plan: SparkPlan, sparkSession: SparkSession): String = {
```

Comment:
shall we simply strip these AQE wrapper nodes at the beginning using `AdaptiveSparkPlanHelper.stripAQEPlan`?